src/cluster.bootstrap/Akka.Management.Cluster.Bootstrap/Internal/BootstrapCoordinator.cs

```diff
@@ -206,6 +206,16 @@ protected override void PreStart()
             StartPeriodicDecisionTimer();
         }
 
+        protected override void PostStop()
+        {
+            Timers.CancelAll();
+            foreach (var child in Context.GetChildren())
+            {
+                child.Tell(PoisonPill.Instance);
```

Comment:
Make sure that all probe actors are told to shut down when the coordinator is shutting down